code/upaya/README.md

```diff
@@ -1,34 +1,66 @@
 upaya
-================
-The recognition of Identity is an often misunderstood concept in our Democracy. 
+=====
+
+The recognition of Identity is an often misunderstood concept in our Democracy.
 
 
 ### Dependencies
 
-- Ruby 2.0+ (try [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/))
+- Ruby 2.0+ (try [rbenv](https://github.com/rbenv/rbenv) or
+  [rvm](https://rvm.io/))
 - Bundler (`gem install bundler`)
+
+### Mac
+
 - [Vagrant](https://www.vagrantup.com/downloads.html)
 - [Virtualbox](https://www.virtualbox.org/wiki/Downloads) or [VMware Fusion](https://www.vmware.com/products/fusion)
 
-Or, try the [Laptop](https://github.com/18F/laptop) script for setting up a slick development environment.
+Or, try the [Laptop](https://github.com/18F/laptop) script for setting up a
+slick development environment.
 
 ### Getting Started
-##### :construction: VM environment coming soon! :construction: 
+
+Run the follow command setup the environment:
```

Comment:
Minor change: the word `to` is missing between `command` and `setup`, and `setup` should be `set up`. The former is a noun, not a verb 😄